Products/Zuul/facades/devicefacade.py

```diff
@@ -953,9 +954,12 @@ def getComponentGraphs(self, uid, meta_type, graphId, allOnSame=False):
             return []
 
         if allOnSame:
-            # ZEN-26498 identify by name to match individual charts
+            # ZEN-26498 ZEN-27126 get docker legends to match individual graphs
             for comp in components:
-                comp.id = comp.name()
+                if (hasattr(comp,'docker_host')):
```

Comment:
The other devices seem to provide the proper objects when grouped into a multigraph -- only Docker containers (via the Docker ZenPack) has been shown to behave this way.